test/controllers/album.test.js

```diff
@@ -67,7 +67,7 @@ describe('Albums Controller', () => {
       it('should respond with photos information', done => {
         nock(config.common.api.albumsApiUrl)
           .get('/photos?albumId=1')
-          .reply(200, '../mocks/photosResponse.json', {
+          .replyWithFile(200, `${process.cwd()}/test/mocks/photosResponse.json`, {
```

Comment:
Why do you change this? You can require this as a dependency 

```
const mockData = require('../mocks/photosResponse.json');
```

Reply:
It doesn't work with just reply. If you use just reply I noticed if you use reply it just responds with a text.

Reply:
This doesn't work?
```
const mockData = require('../mocks/photosResponse.json');

..other tests

it('should respond with photos information', done => {
        nock(config.common.api.albumsApiUrl)
          .get('/photos?albumId=1')
          .reply(200, mockData, { ...

```